sql/catalyst/src/main/scala/org/apache/spark/sql/errors/SqlScriptingErrors.scala

```diff
@@ -43,6 +44,40 @@ private[sql] object SqlScriptingErrors {
       messageParameters = Map("endLabel" -> endLabel))
   }
 
+  def sqlScriptingNotEnabled(origin: Origin): Throwable = {
+    new SqlScriptingException(
+      origin = origin,
+      errorClass = "UNSUPPORTED_FEATURE.SQL_SCRIPTING_NOT_ENABLED",
+      cause = null,
+      messageParameters = Map("sqlScriptingEnabled" -> SQLConf.SQL_SCRIPTING_ENABLED.key))
```

Comment:
Please, follow the existing convention and quote the SQL config name using `toSQLConf()`